### ISSUE TITLE
Initiate - general approach is negative

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-dynamic-planning#11]
Overall, I think our approach is fairly dismal.  it assumes that there is little interest in accessibility which I think is an outdated understanding. We make accessibility seem like a long dull slog, which it should not be. I think we are in a place of increased interest but a general lack of understanding and much confusion about how to proceed.

A more effective approach in my opinion would be one that is "brighter," that recognizes the value of accessibility as a design tool that is gaining credibility but around which there is a rocky road based on much confusion, lack of understanding, and still considerable fear. Our job is to smooth the path and quell the fear. Accessibility is most often a design challenge and the more it is integrated into existing practice, the more smoothly the process is for everyone. 

Examples of the dismal approach and suggestions for rewording:

From Basics: 
- "help convince others of the need" change to "help others understand the value/benefits/etc"
- "What is accessibility?" current paragraph is too dull to effectively engage. How about something like "Web accessibility fulfills the basic promise of the web - making information and communication readily available to all people regardless of barriers in geography, language, or disability.  Good web designers understand this as well as the unanticipated benefits that accrue to organizations that integrate accessibility into their practice - benefits that include better user experiences on mobile devices and older technology often used by aging populations. 

From Review
- does "scale of the challenge" sound like climbing Mt Everest?

From Raise Awareness
- overall tone is just the facts about how unaware people are and statements like "general intro...will benefit everyone" is not substantiated. How will it help everyone? 
  Suggested approach is more lively, emphasizing what is to be gained. (This is too wordy, but I am describing an approach) Something like "Consider your specific audience as you raise awareness of the benefits of web accessibility.  Management may be most interested in risk management but designers are likely to get excited by understanding and  solving the design challenges of accessibility. Developers will be engaged to evolve their programming skills by integrating the modern elements of HTML5, javascript and ARIA that address accessibility. Marketing may be interested in the stories of users and demonstrating corporate responsibility.  etc" 

From Business case
- again I feel like we are slogging through. "business case will be used to persuade stakeholders..." Why do they need persuasion? what if I don't persuade them? prediction: dismal experience.  
  How about "the effective business case demonstrates why and how web accessibility aligns and can be integrated into organizational strategies and goals."

From Gather Support
- here we are "persuading" people and groups again. I don't think we want to frame this as a need for persuasion, but as an opportunity for enlightenment, understanding, improved communication strategies, better, more modern skill sets, etc. Help people see they are missing the boat, not that they have to do this dutiful thing because of legal obligation or government mandate. Accessibility is _super cool,_ is foundational to the very meaning of the web and if they don't get on this bandwagon, they will be a schmuck designer, developer, marketer, PM, whatever.

**Comment by iadawn** [comment]
I have gone through the items you flagged and made some changes. See what you think. You might like to look at the changes in Gather Support, as I think the third sentence in the second paragraph might be lacking something:

> Encourage interest in these groups by exploring how accessibility has broad benefits. For example, individuals can improve their skills, organizations improve their websites, and people with disabilities have a better online experience.

I will keep the happy feeling in mind when going through the other sections!
